examples/cpp/CMakeLists.txt

```diff
@@ -62,6 +62,9 @@ add_example(NAME pdxserializer
 add_example(NAME put-get-remove
     SOURCE main.cpp)
 
+add_example(NAME remote-function
```

Comment:
this example should be called `function-execution` as that is accurate and matches the docs too